lib/UserNotificationConfigHandler.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ *
+ * @author Juan Pablo Villafáñez <jvillafanez@solidgear.es>
+ * @copyright Copyright (c) 2018, ownCloud GmbH
+ * @license GPL-2.0
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\PasswordPolicy;
+
+use OCP\IConfig;
+use OCA\PasswordPolicy\Db\OldPassword;
+
+class UserNotificationConfigHandler {
+	const DEFAULT_EXPIRATION_FOR_NORMAL_NOTIFICATION = 30 * 24 * 60 * 60;  // 30 days
```

Comment:
Another place where having the "magic" ``24 * 60 * 60`` somewhere common will help for the future.

Reply:
I think this is fine because you see clearly the conversion. The other option is to write directly the number of seconds, but it will be a random number for a lot of people and we'll need to trust the comment.

Reply:
No problem - I agree that writing something like ``30*86400`` is not a good thing!